kernel-python/declarativewidgets/widget_visualizations.py

```diff
@@ -3,12 +3,14 @@
 
 from IPython.core.display import display, HTML
 
-def explore(df, channel='default', selection_var=None):
+def explore(df, channel='default', selection_var=None, selection_as_object=True):
```

Comment:
I think `selection_as_object` should allow being unset. If it needs to be set to either `True` or `False`, it should match the default for the property in the elements. I think that is `False`.